.github/workflows/ci.yaml

```
@@ -313,3 +313,40 @@ jobs:
         if: ${{ steps.filter-images.outputs.kube-webhook-certgen == 'true' }}
         run: |
           cd images/kube-webhook-certgen && make test test-e2e
+
+  rancher-validate: 
+    runs-on: ubuntu-latest
+    env:
+      PLATFORMS: linux/amd64
+    container:
+      image: rancher/dapper:v0.5.3
+      env:
+        GIT_IN_DAPPER: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: validate
+        run: |
+          dapper validate
+
+  rancher-build:
+    needs: rancher-validate
+    runs-on: ubuntu-latest
+    env:
+      PLATFORMS: linux/amd64
+    container:
+      image: rancher/dapper:v0.5.3
+      env:
+        GIT_IN_DAPPER: true
+    steps:
+      - name: Install git
+        run: |
+          apk add git
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: build
+        run: |
+          dapper build
```

nit: add a blank line at the end of the file
```suggestion
          dapper build

```